metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -303,6 +320,10 @@ def get_inspectors(self) -> Iterable[Inspector]:
             inspector = inspect(conn)
             yield inspector
 
+    def get_db_schema(self, dataset_identifier: str) -> Tuple[Optional[str], str]:
```

Comment:
iirc we had a PR about athena being two vs three tier at some other point - what was the ultimate conclusion there?